pages/button-dropdown/item-element.permutations.page.tsx

```diff
@@ -17,6 +17,7 @@ const permutations = createPermutations<ItemProps>([
     highlighted: [false, true],
     last: [false, true],
     hasCategoryHeader: [false],
+    hasExpandableGroups: [false],
```

Comment:
Are these additions necessary? Won't the component default to false anyway?

Reply:
I made the property required to not forget to pass it on accident